Gaijinizer patch/Map154.txt

```diff
@@ -339,10 +341,9 @@ and forth again with you again#MTLed
 具現種に成っていないとしても、会いたい時は
 そこで会えばいいわけじゃ、ここで戦う『得』がない。
 > CONTEXT: Map154/events/4/pages/1/41/Dialogue/enemy17
-Mio is already in my room at Nushi.
-If you don't have tranformed species, when you
-want to meet. There is no way to meet them, there
-is no "profit" to fight here.#MTLed
+I am already in My Room. If you wish to meet, 
+then you can meet me there. There's no benefit
+for me in fighting you here.#CheckLater
 > END STRING
```

Comment:
Note?

Reply:
I think this one was because there's a line in there saying something to the effect of "even though I'm no longer a Spirit, if you wish to meet, then you can meet me in there." And it didn't really translate well.

Reply:
Okay... some minor revisions then:
```
I am already in My Room. If you wish to see me, 
then you can meet me there. There's no benefit
for me in fighting you here.
```